src/ansys/mapdl/core/_commands/database/components.py

```diff
@@ -74,7 +74,7 @@ def cm(self, cname="", entity="", **kwargs):
         >>> mapdl.asel('S', 'LOC', 'Y', loc)
         >>> mapdl.cm('PRES_A', 'AREA')
         """
-        command = "CM,%s,%s" % (str(cname), str(entity))
+        command = f"CM,{str(cname)}, {str(entity)}"
```

Comment:
Surely you know but you don't need to use ``str()`` when using f-strings I believe ;)

```python
        command = f"CM,{cname}, {entity}"
```

Reply:
Uh yeah, I have gone too fast through the PR. F-strings already do that for you @clatapie